pithos/plugins/notify.py

```diff
@@ -45,6 +45,18 @@ def on_prepare(self):
         if 'action-icons' in caps:
             self.notification.set_hint('action-icons', GLib.Variant('b', True))
 
+        # GNOME Shell 3.20 or higher has bultin MPRIS functionality that makes
+        # persistent song notifications redundant.
+        has_built_in_mpris = False
+        version = server_info['version'].split('.')
```

Comment:
I've never seen it fail. If it had we would have got bug reports about GioNotify I'd think since it turns the list of the values into the dict.